autogen/oai/client.py

```diff
@@ -281,6 +281,14 @@ def _completions_create(self, client, params):
             for chunk in completions.create(**params):
                 if chunk.choices:
                     for choice in chunk.choices:
+                        # If the ChoiceDelta returns with content is None and a ChoiceDeltaFunctionCall value
+                        # We set streaming to false, as this message is calling a function, and call completions.create
+                        # So that we are sending a regular (not streaming) chat completion request
+                        if choice.delta.function_call and not choice.delta.content:
+                            params = params.copy()
+                            params["stream"] = False
+                            response = completions.create(**params)
+                            return response
                         content = choice.delta.content
```

Comment:
In these lines, "completions.create()" is invoked twice, resulting in two requests to OpenAI. The solution involves managing the scenario when "choice.delta.function_call" is not None and extracting the data to stream, which is notably more intricate.

Reply:
I think I have resolved these issues in #1118